pkg/models/plan_ops_eval_and_learning.go

```diff
@@ -157,14 +157,14 @@ const (
 	DataIncremental DataFullTimeOrIncrementalType = "INCREMENTAL"
 )
 
-// QualityPerformanceImpactType represents the possible QUALITYPERFORMANCEIMPACTTYPE options
-type QualityPerformanceImpactType string
+// YesNoOtherType represents the possible YESNOOTHERTYPE options
```

Comment:
This could maybe go in the shared enums file? But it think it also makes sense to leave it here because it is technically not being shared at this moment in time.